TestEtcdSourceExistingBoundPods flaky

Welcome to travis' surprising timing...

```
--- FAIL: TestEtcdSourceExistingBoundPods (0.04 seconds)
	<autogenerated>:29: creating /registry/nodes/machine/boundpods, index 0
	<autogenerated>:29: returning /registry/nodes/machine/boundpods: &etcd.Response{Action:"", Node:(*etcd.Node)(0xc208005080), PrevNode:(*etcd.Node)(nil), EtcdIndex:0x0, RaftIndex:0x0, RaftTerm:0x0} <nil>
	etcd_test.go:75: Expected update, timeout insteam
```

--- a/pkg/kubelet/config/etcd_test.go
+++ b/pkg/kubelet/config/etcd_test.go
@@ -71,8 +71,8 @@ func TestEtcdSourceExistingBoundPods(t *testing.T) {
 			update.Pods[1].ObjectMeta.Name != "bar" {
 			t.Errorf("Unexpected update response: %#v", update)
 		}
-	case <-time.After(2 * time.Millisecond):
-		t.Errorf("Expected update, timeout insteam")
+	case <-time.After(200 * time.Millisecond):
+		t.Errorf("Expected update, timeout instead")
 	}
 }
 

